docs/tutorial/creating_main.rst

```diff
@@ -0,0 +1,25 @@
+
+The __main__ File
+---------------------
+
+Every program needs a main. Therefore, the first file you create should be called::
+
+	__main__.py
+
+For a Quickstart, you can copy the following code, taken from the `'Ball and Beam' example`_ . 
```

Comment:
Basically all self references in the docs should be relative (see comment above)